hz: 10
local_goal_dist: 4.0
lap_num: 1